app/src/main/java/com/gh4a/BaseActivity.java

```diff
@@ -112,6 +112,8 @@
     private Animator mHeaderTransition;
     private final Handler mHandler = new Handler();
 
+    private final List<Disposable> mDisposeOnStop = new ArrayList<>();
```

Comment:
Apparently you can no longer use it after calling `dispose`. But it might be possible to use clear `instead` which based on [documentation](http://reactivex.io/RxJava/javadoc/io/reactivex/disposables/CompositeDisposable.html#clear--) should dispose items.